pkg/test/requester/retries_test.go

```diff
@@ -132,13 +132,10 @@ func (s *RetriesSuite) SetupSuite() {
 	}
 	ctx := context.Background()
 
-	requesterConfig, err := node.NewRequesterConfigWith(
-		node.RequesterConfigParams{
-			NodeRankRandomnessRange: 0,
-			OverAskForBidsFactor:    1,
-		},
-	)
```

Comment:
No ides how this ever worked before `NodeRankRandomnessRange` cannot be `0` or the code that uses it panics, somehow it never panicked, meaning the setting of these values were likely ineffectual